<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="filename_layout" stretch="0,1">
      <item>
       <widget class="QPushButton" name="choose_file">
        <property name="text">
         <string>Choose file...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="source_file_path">
        <property name="text">
         <string>PLACEHOLDER: filename</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="dir_layout" stretch="0,1">
      <item>
       <widget class="QPushButton" name="choose_directory">
        <property name="text">
         <string>Choose dir...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="destination_dir_path">
        <property name="text">
         <string>PLACEHOLDER: dirpath</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="error">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ff0000;&quot;&gt;Auto saving is stopted. Choose right file or dir to start auto saving.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout"/>
    </item>
    <item>
     <widget class="QTableWidget" name="table"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
